### PR TITLE
Update create-a-phone-system-call-queue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Minor clarifications to indicate music on hold is for in queue calls only.
Added note that default music is free of any royalties payable by the organization.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-call-queue.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Call queues provide:
 
 - A greeting message.
 
-- Music while people are waiting on hold.
+- Music while people are waiting on hold in queue.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:36]:
```suggestion
- Music while people are waiting on hold in a queue.
```

File: Teams/create-a-phone-system-call-queue.md [hunk 2]
```diff
@@ -53,14 +53,15 @@ To set up a call queue, in the Teams admin center, expand **Voice**, click **Cal
 
 3. Choose a language. This language will be used for system-generated voice prompts and voicemail transcription (if you enable them).
 
-## Greetings and  hold music
+## Greetings and hold music
 
 Specify if you want to play a greeting to callers when they arrive in the queue. You must upload an MP3, WAV, or WMA file containing the greeting that you want to play.
 
-Teams provides default music to callers while they are on hold. If you want to play a specific audio file, choose **Play an audio file** and upload an MP3, WAV, or WMA file.
+Teams provides default music to callers while they are on hold in queue. If you want to play a specific audio file, choose **Play an audio file** and upload an MP3, WAV, or WMA file.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:60]:
```suggestion
Teams provides default music to callers while they are on hold in a queue. If you want to play a specific audio file, choose **Play an audio file** and upload an MP3, WAV, or WMA file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits, please.

File: Teams/create-a-phone-system-call-queue.md [hunk 3]
```diff
@@ -53,14 +53,15 @@ To set up a call queue, in the Teams admin center, expand **Voice**, click **Cal
...
 
 > [!NOTE]
-> The uploaded recording can be no larger than 5 MB..
+> The uploaded recording can be no larger than 5 MB.
+> The default music supplied by in Teams Call Queue is free of any royalties payable by your organization.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:64]:
```suggestion
> The default music supplied in Teams Call Queue is free of any royalties payable by your organization. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**